src/v3/src/transformer/phone/phoneEnrollmentTransformer.ts

```diff
@@ -76,37 +77,62 @@ export const transformPhoneEnrollment: IdxStepTransformer = ({ formBag, transact
     },
   };
 
-  const stepper: StepperLayout = {
-    type: UISchemaLayoutType.STEPPER,
-    elements: [
-      {
-        type: UISchemaLayoutType.VERTICAL,
-        elements: [
-          smsInfoTextElement,
-          methodTypeElement,
-          phoneNumberElement,
-          smsStepSubmitButton,
-        ],
-      },
-      {
-        type: UISchemaLayoutType.VERTICAL,
-        elements: [
-          voiceInfoTextElement,
-          methodTypeElement,
-          phoneNumberElement,
-          voiceStepSubmitButton,
-        ],
-      },
-    ],
+  const phoneOptionLayouts: { [key:string]: UISchemaLayout | StepperLayout } = {
+    sms: {
+      type: UISchemaLayoutType.VERTICAL,
+      elements: [
+        smsInfoTextElement,
+        phoneNumberElement,
+        smsStepSubmitButton,
+      ],
+    },
+    voice: {
+      type: UISchemaLayoutType.VERTICAL,
+      elements: [
+        voiceInfoTextElement,
+        phoneNumberElement,
+        voiceStepSubmitButton,
+      ],
+    },
```

Comment:
`uischema.elements` looks to already be a UISchemaLayout, it seems like you can just append these elements directly to uischema.elements instead of adding a new layout.